enterprise_access/apps/content_assignments/tasks.py

```diff
@@ -509,6 +513,9 @@ def send_email_for_new_assignment(new_assignment_uuid):
         'learner_portal_link',
         'action_required_by_timestamp'
     )
+
+    logger.info(f'Braze properties for assignment {assignment.uuid}: {braze_trigger_properties}')
```

Comment:
[inform] Upon further investigation, it looks like we _do_ already log the `braze_trigger_properties` [here](https://github.com/openedx/enterprise-access/blob/b756728a4acf73e92df3cdb945a42251bf3dc43a/enterprise_access/apps/content_assignments/tasks.py#L119-L125). We might not actually need to include this log line as a result?

The `[get_start_date]` logs will give us the additional missing information though, around what source metadata is used to derive the incorrect start date.